src/playbooks/with_build/upgrade_vcins.yml

```diff
@@ -0,0 +1,20 @@
+- name: Run VCIN inplace upgrade if eligible
+  import_playbook: "upgrade_vcins_inplace.yml"
+
```

Comment:
You should add a check for predeploy here to cover if the user calls upgrade_vcins directly.  The hosts should be "vcin_upgrade_sa_node" as to not trigger on inplace upgrade.

Reply:
Done